cypress/e2e/clue/branch/student_tests/drawing_tool_spec.js

```diff
@@ -234,11 +234,22 @@ context('Draw Tool Tile', function () {
       });
       it("deletes rectangle drawings", () => {
         drawToolTile.getDrawTile().click();
-        for (let i=0; i<6; i++) {
+        // delete the first 4 with the toolbar button
+        for (let i=0; i<4; i++) {
           drawToolTile.getDrawToolSelect().click();
           drawToolTile.getRectangleDrawing().first().click({force:true, scrollBehavior: false});
           drawToolTile.getDrawToolDelete().click();
         }
+        // Delete with backspace key
+        drawToolTile.getDrawToolSelect().click();
+        drawToolTile.getRectangleDrawing().first().click({force:true, scrollBehavior: false});
```

Comment:
Just curious about our use of force:true in these tests (I just read about it now). Is that something we do because we've found the tests to be flaky because elements aren't in view, visible, etc?

Reply:
We don't have a standard approach. I do try to only use them when necessary. However you'll notice that others will use them most of the time.

I don't remember exactly, but I believe in this case the force was initially added because part of the SVG object being clicked was partly off the screen. I think in that case cypress wouldn't allow a click on it. Perhaps with the `scrollBehavior: false` it is no longer necessary. But given the strange scroll behavior issue it seems good to keep the force until we fix that scroll issue. If you didn't see there is a comment at the top of the file about the scrollBehavior issue.